aptos-move/framework/aptos-framework/sources/staking_contract.spec.move

```diff
@@ -237,6 +287,17 @@ spec aptos_framework::staking_contract {
         // TODO: set because of timeout
         pragma verify = false;
         include GetStakingContractAmountsAbortsIf;
+
+        let pool_address = staking_contract.pool_address;
```

Comment:
The "verify" pragma is set to false. Are these specs tested?

Reply:
yes, it has been tested and it passed, remove thhe verify